indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -571,6 +574,26 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_avg_per_question_per_track(self):
+        ratings = {}
+        abstract_ratings = (AbstractReviewRating.query
+                            .join(AbstractReviewRating.review)
+                            .join(AbstractReviewRating.question)
+                            .options(contains_eager('review'), contains_eager('question'))
+                            .filter(AbstractReview.abstract == self,
+                                    ~AbstractReviewQuestion.is_deleted)
+                            .all())
+        for rating in abstract_ratings:
+            if rating.question.no_score:
+                continue
+            ratings.setdefault(rating.review.track_id, {})
+            question_avg_rating = ratings[rating.review.track_id].get(rating.question)
+            if question_avg_rating:
+                ratings[rating.review.track_id][rating.question] = (question_avg_rating + rating.value) / 2
```

Comment:
I'd rather build at list with the scores and then divide the sum of that list by the length of it.